src/js/game/LevelMVC/LevelView.js

```diff
@@ -467,7 +467,9 @@ module.exports = class LevelView {
     let blockIndex = (this.yToIndex(position[1])) + position[0];
     let block = this.actionPlaneBlocks[blockIndex];
     let animationName = open ? "open" : "close";
-    this.onAnimationEnd(block.animations.play(animationName, 60), () => {
+    const animation = this.playScaledSpeed(block.animations, animationName);
+    this.onAnimationEnd(animation, () => {
+      animation.updateCurrentFrame();
```

Comment:
is this (calling `updateCurrentFrame` on animation end) something we should be doing in general?

Reply:
TBH I'm not sure.  It's an undocumented API...  Until this specific issue is fixed upstream I think we need this to fix the door animation.  Other animations loop or disappear when complete.

I'm tempted to only patch this here, then revisit if we see another case of this bug with a different animation.  What do you think?

Reply:
Also not all animations are connected to an `onAnimationEnd` event.  Is there one place we can add this to fix this class of issues?

Reply:
There unfortunately is not one place we could go to fix this kind of issue, but it might be nice at some point to add one so our animations can in general be a little more standardized.

I think fixing this just here makes sense for now